packages/gestalt/src/Popover.js

```diff
@@ -9,18 +9,69 @@ type Size = 'xs' | 'sm' | 'md' | 'lg' | 'xl' | 'flexible' | number;
 type IdealDirection = 'up' | 'right' | 'down' | 'left';
 
 type Props = {|
+  /**
+   * The reference element, typically [Button](/Button) or [IconButton](/IconButton), that Popover uses to set its position.
+   */
   anchor: ?HTMLElement,
+  /**
+   * The content shown in Popover.
+   */
   children?: Node,
-  color?: Color,
+  /**
+   * The background color of Popover. See the [color and caret](#Color-and-caret) variant to learn more.
+   *
+   * Default: 'white'
+   *
+   */
+  color?: 'blue' | 'orange' | 'red' | 'white' | 'darkGray',
+  /**
+   * Callback fired on keyboard navigation (keydown event).
+   */
   handleKeyDown?: (event: SyntheticKeyboardEvent<HTMLElement>) => void,
+  /**
+   * Unique id to identify each Popover. Used for [accessibility](#ARIA-attributes) purposes.
+   */
   id?: string,
-  idealDirection?: IdealDirection,
+  /**
+   * Specifies the preferred position of Popover relative to its anchor element. See the [ideal direction](#Ideal-direction) variant to learn more.
+   */
+  idealDirection?: 'up' | 'right' | 'down' | 'left',
+  /**
+   * Callback fired when Popover requests to be closed. Must be used to control Popover’s on/off display state.
+   */
   onDismiss: () => void,
+  /**
+   * Properly positions Popover relative to its anchor element. Set to false when used within [Layer](/Layer. See the [with Layer](#With-layer) variant to learn more.
```

Comment:
@AlbertCarreras looks a bit broken, I think there is a square bracket missing at the end of `[Layer](/Layer`
![image](https://user-images.githubusercontent.com/127199/132590753-f938584d-6985-48d2-ad24-ced5024331d3.png)